### PR TITLE
Support mysql 8 server

### PR DESCRIPTION
**Pull request by Romfos**
Changes in this PR:
- Updated mysql.exe client that will be support mysql 8 
- Refactroing project structure

---
This change is [Reviewable](https://reviewable.io/reviews/mangosvb/databasezero/2)

### CODE REVIEW

**Comment by billy1arm** [comment]
something to bear in mind.... mysql 8.0 support needs to be done in the code before the database support can be applied.

There are also several test scenarios that must be passed before the pr can be approved:

1) MySql server running 5.7. local pc has no mysql installed
2) MySql server running 8.0, local pc has no mysql installed

Both of the above scenarios are typical when users are installing the database or updates to the server remotely

3) MySql server running 5.7. local pc has MySql 5.7 installed
4) MySql server running 8.0, local pc has MySql 8.0 installed

This is why the mysql.exe and mysqldump.exe are in the tools folder taken from 5.7.
- We still have not solved this over on the c++ mangos database repos yet as placing the above files from 8.0 into the tools folder doesn't work as they now require additional dll files when 8.0 is not installed locally. I believe they need some ssl related dll's placed in there as well, but have not had the time to investigate further.

File: Install_World.bat [hunk 1]
```diff
@@ -64,11 +64,11 @@ goto :eof
 
 :world
 if %quick% == off echo.
-if %quick% == off echo This will wipe out your current World database and replace it.
+if %quick% == off echo This will wipe out your current Word database and replace it.
```
Comment (billy1arm) [inline Install_World.bat]: Typo: should be World

File: Install_World.bat [hunk 2]
```diff
@@ -64,11 +64,11 @@ goto :eof
...
 if %quick% == off set /p yesno=Do you wish to continue? (y/n) 
 
 echo.
-echo Importing World database
+echo Importing Word database
```
Comment (billy1arm) [inline Install_World.bat]: Typo: should be World

File: Install_World.bat (hunk 1 above)
Comment (Romfos) [reply Install_World.bat]: fixed

File: Install_World.bat (hunk 2 above)
Comment (Romfos) [reply Install_World.bat]: fixed

**Comment by Romfos** [comment]
@billy1arm I don't have any locally install mysql server. I'm using mysql in docker for test porpose.
Current mysql.exe (from mysql subfolder) tested for mysql 5.6 and 8.0 and works fine.

I think it is safe to merge

**Comment by billy1arm** [comment]
Thanks for your contribution

**Pull request merged**